include/wlr/types/wlr_surface.h

```diff
@@ -91,8 +91,8 @@ struct wlr_surface {
 	/**
 	 * The surface's buffer, if any. A surface has an attached buffer when it
 	 * commits with a non-null buffer in its pending state. A surface will not
-	 * have a buffer if it has never committed one, has committed a null buffer,
-	 * or something went wrong with uploading the buffer.
```

Comment:
If the first attached buffer failed during import/upload, this can still happen.

Reply:
Right. Although, if that were to happen, the surface would've been unmapped already, so there's no reason to have those checks anyway.
Should I change it to "something went wrong with uploading the *first* buffer"?